hak/keytar/hak.json

```diff
@@ -0,0 +1,11 @@
+{
+    "scripts": {
+        "check": "check.js",
+        "build": "build.js"
+    },
+    "prune": "native",
```

Comment:
This line can be removed: there is no `native` directory in keytar - its build dir is `build` but it's only created at build time so doesn't need to be removed from the distribution copy.